tests/wpt/metadata/resource-timing/idlharness.any.js.ini

```diff
@@ -82,9 +76,6 @@
   [PerformanceResourceTiming interface: resource must inherit property "transferSize" with the proper type]
     expected: FAIL
 
-  [PerformanceResourceTiming interface: resource must inherit property "domainLookupStart" with the proper type]
-    expected: FAIL
```

Comment:
Restore this particular failure (just these two lines) to this file and the test should stop reporting an unexpected failure.

Reply:
Thank you. Done